core/primitives/src/runtime/config_store.rs

```diff
@@ -36,6 +36,7 @@ static CONFIG_DIFFS: &[(ProtocolVersion, &str)] = &[
     (61, include_config!("61.yaml")),
     (62, include_config!("62.yaml")),
     (63, include_config!("63.yaml")),
+    #[cfg(not(feature = "sandbox"))]
```

Comment:
This doesn’t seem like a correct approach. What if somebody adds another parameter change to this version, even if in the future releases of the protocol? This is also a departure from how sandbox functionality changes have been implemented in the past, which is feature gating specific areas of the implementation, rather than configuration.